Sources/Containerization/VirtualMachineAgent.swift

```diff
@@ -66,6 +66,7 @@ public protocol VirtualMachineAgent: Sendable {
     func down(name: String) async throws
     func addressAdd(name: String, address: String) async throws
     func routeAddDefault(name: String, gateway: String) async throws
+    func routeAddLink(name: String, address: String, gateway: String, srcAddr: String) async throws
```

Comment:
You'll need to add an unsupported entry to the bottom like what is done for `closeProcessStdin1 and the others